projects/Mallard/src/helpers/files.ts

```diff
@@ -283,3 +284,13 @@ export const downloadAndUnzipIssue = (
             console.log('Download error: ', error)
         })
 }
+
+export const clearPastSevenDaysIssues = async () => {
```

Comment:
Took me a while to figure out the wording on this function name, maybe `clearOldIssues`?